src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -72,14 +74,14 @@ public void setConnections(MainController mainController, ModelManager modelMana
      */
     @FXML
     private void handleDeletePerson() {
-        int selectedIndex = personList.getSelectionModel().getSelectedIndex();
+        int selectedIndex = personListView.getSelectionModel().getSelectedIndex();
         if (selectedIndex >= 0) {
-            Person tmpPerson = personList.getItems().get(selectedIndex);
-            mainController.getStatusBarHeaderController().postStatus(
-                                                    new PersonDeletedStatus(tmpPerson));
-            personList.getItems().get(selectedIndex).setIsDeleted(true);
+            ObservableViewablePerson target = personListView.getItems().get(selectedIndex);
```

Comment:
Variable name `target` is confusing. Target of what; deletion? Maybe `personToBeDeleted` or something more descriptive.